### DISCUSSION TITLE
How to specify amount of lines to be printed of Code mobject?

### DISCUSSION THREAD

**Discussion by Rassska** [3b1b/manim#1773] [Q&A]
Hi, there!
Is it possible to specify a certain amount of lines or range to be printed?
Well, let's say, I do wanna write some code in the middle of the background rectangle, after that add up some more lines in different part of my window?

By now, I do have smth like that:

```
cppCode = Code(
            "defaultCode.cpp",
            tab_width=4,
            line_spacing = 1,
            line_no_buff = 1,
            font_size = 14,
            stroke_width = 0,
            background_stroke_width=1,
            background_stroke_color=WHITE,
            style = Code.styles_list[37], #37, 41
            insert_line_no=False,
            background="window",
            font = "Ubuntu",
            language="cpp",
        ).next_to(cppCodeLabel, direction = DOWN, buff = 0.3)
        self.play(Write(cppCode))
```

**Comment by thehassantahir** [comment]
This might help you [Stack](https://www.codegrepper.com/code-examples/cpp/print+all+number+between+a+and+b+in+c%2B%2B)

**Reply by Rassska** [reply]
Hi, thanks for your reply, but unfortunately you might misunderstood what i wanna get. 

Basically, I do have a .cpp file and now I want to render only the half of that .cpp file, and after some time maybe another half etc.

**Comment by thehassantahir** [comment]
Hey Rassska,
Nice to see a follow up! Now i got your point.
why don't you comment half of your .cpp file? and uncomment it later when want to render that part?

as you mentioned `Is it possible to specify a certain amount of lines or range to be printed?`

lets suppose:
you have 3 lines 
1. 
2.
3.
and you want dont want to render the 2. line just comment it if it is not disturbing your function.
